### PR TITLE
Recommend TensorFlow release based on CUDA present in environment

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
all green, ready to be merged

File: thoth/adviser/sieves/tensorflow/tf_cuda.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+
+@attr.s(slots=True)
+class TensorFlowCUDASieve(Sieve):
+    """A sieve that makes sure the right TensorFlow version is used when CUDA is present in the runtime environment.
+
+    See supported matrix at https://www.tensorflow.org/install/source#linux
+    """
+
+    _MESSAGE = f"Recommended TensorFlow that supports CUDA present in the runtime environmen - see {jl('tf_cuda')}"
```
Comment (harshad16) [inline thoth/adviser/sieves/tensorflow/tf_cuda.py:52]:
```suggestion
    _MESSAGE = f"Recommended TensorFlow that supports CUDA present in the runtime environment - see {jl('tf_cuda')}"
```

**Review by harshad16 — APPROVED** [review]
lgtm

**Pull request merged**